### DISCUSSION TITLE
Changing Language in Component

### DISCUSSION THREAD

**Discussion by elnasnato** [tasteui/tasteui#708] [Q&A] [answered]
I want to know how to change the language in the component, for example in the x-table  component there is a label "quantity", I want to be able to change it or it has two languages. I don't know if I missed it in the documentation.

**Comment by devajmeireles** [comment, marked as answer]
Hey, @elnasnato ! All the TSUI component text strings are designed using the Laravel's translation system. If the table component has a `quantity` text, you can interact with it by publishing the TSUI translations files: https://tallstackui.com/docs/translation
